staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -94,6 +94,11 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *v1beta
 		}
 	}
 
+	// Currently dispatcher only supports `v1beta1` AdmissionReview
+	if !util.HasAdmissionReviewVersion("v1beta1", h) {
```

Comment:
use constant, and add TODO to make this capable of sending/receiving multiple versions